### PR TITLE
Add packages for vvmd and vvmplayer

### PR DESCRIPTION
**Pull request by ArenM**

### CODE REVIEW

File: PKGBUILDS/danctnix/vvmd/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+package() {
+  meson install -C build --destdir "$pkgdir"
+
+  install -Dm644 "vvmd.service" "$pkgdir/usr/lib/systemd/user/vvmd.service"
+
+  mkdir -p "$pkgdir/usr/lib/systemd/user/default.target.wants"
+  ln -sfv "/usr/lib/systemd/user/vvmd.service" \
+    "$pkgdir/usr/lib/systemd/user/default.target.wants/vvmd.service"
```
Comment (Danct12) [inline PKGBUILDS/danctnix/vvmd/PKGBUILD:32]: Could this be moved to a post install script?

File: PKGBUILDS/danctnix/vvmd/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pkgname=vvmd
+pkgver=0.6.1
+pkgrel=1
+pkgdesc="Visual Voicemail Daemon"
+url="https://gitlab.com/kop316/vvmd"
+arch=('x86_64' 'aarch64')
+license=("GPL2")
+depends=("mobile-broadband-provider-info" "libmm-glib" "libphonenumber")
+makedepends=("git" "meson")
+source=("$pkgname.tar.gz::https://gitlab.com/kop316/vvmd/-/archive/$pkgver/vvmd-$pkgver.tar.gz"
```
Comment (Danct12) [inline PKGBUILDS/danctnix/vvmd/PKGBUILD:11]:
```suggestion
source=("$pkgname-$pkgver.tar.gz::https://gitlab.com/kop316/vvmd/-/archive/$pkgver/vvmd-$pkgver.tar.gz"
```

File: PKGBUILDS/danctnix/vvmplayer/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+pkgname=vvmplayer
+pkgver=1.3
+pkgrel=1
+pkgdesc="VVM Player is a front end GUI to display, play, and delete visual voicemails"
+url="https://gitlab.com/kop316/vvmplayer"
+arch=('x86_64' 'aarch64')
+license=("GPL3")
+depends=("gtk3" "libhandy" "gstreamer" "evolution-data-server" "callaudiod" "vvmd")
+makedepends=("git" "meson")
+source=("$pkgname.tar.gz::https://gitlab.com/kop316/vvmplayer/-/archive/$pkgver/vvmplayer-$pkgver.tar.gz")
```
Comment (Danct12) [inline PKGBUILDS/danctnix/vvmplayer/PKGBUILD:11]:
```suggestion
source=("$pkgname-$pkgver.tar.gz::https://gitlab.com/kop316/vvmplayer/-/archive/$pkgver/vvmplayer-$pkgver.tar.gz")
```

**Comment by Danct12** [comment]
Other than that, LGTM and build without errors on a clean chroot.

However, I have no way to test this as my phone carrier does not support visual voicemail. Anyone wants to test it?

**Comment by ArenM** [comment]
Updated. I also removed the make dependency on git which was left over from the git version.

File: PKGBUILDS/danctnix/vvmd/PKGBUILD (hunk 1 above)
Comment (ArenM) [reply PKGBUILDS/danctnix/vvmd/PKGBUILD]: Done

File: PKGBUILDS/danctnix/vvmd/PKGBUILD (hunk 2 above)
Comment (ArenM) [reply PKGBUILDS/danctnix/vvmd/PKGBUILD]: Done

File: PKGBUILDS/danctnix/vvmplayer/PKGBUILD (hunk 3 above)
Comment (ArenM) [reply PKGBUILDS/danctnix/vvmplayer/PKGBUILD]: Done

**Comment by Danct12** [comment]
One peek again and LGTM, let's merge this!

**Pull request merged**